livingpark_utils/livingpark_utils.py

```diff
@@ -791,7 +795,7 @@ def make_gif(self, frame_folder: str, output_name: str = "animation.gif") -> Non
         )
         print(f"Wrote {os.path.join(frame_folder, output_name)}")
 
-    def qc_spm_segmentations(self, cohort) -> None:
+    def qc_spm_segmentations(self, cohort, mni_space = False, show_gm = False, show_wm = False, cut_coords=(-28, -7, 17)) -> None:
```

Comment:
default values should be all set to True. With the current default values, nothing is displayed.